frontend/src/dto/CallLogs/CallLogDTOs.ts

```diff
@@ -0,0 +1,23 @@
+export interface CallUpdateDTO {
+  receiverName?: string;
+  date?: string;
+  phoneNumber?: string;
+  description?: string;
+  receiverEmail?: string;
+  action?: string;
```

Comment:
@nasaku898 Although this still works, do you think it's necessary to change it to the correct type? (action should be of type ENUM and date should be of type Date)

Reply:
I agree with the ENUM. For the date, I think it's okay to leave it as a string since what we are sending to the backend is a date but in a string format so I think it is fine.